src/main/java/com/wcc/platform/service/PageService.java

```diff
@@ -0,0 +1,50 @@
+package com.wcc.platform.service;
+
+import com.wcc.platform.domain.cms.pages.FooterPage;
+import com.wcc.platform.domain.cms.pages.LandingPage;
+import com.wcc.platform.repository.PageRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+/** Pages Service. */
+@Service
+public class PageService {
```

Comment:
The previous [PR](https://github.com/Women-Coding-Community/wcc-backend/pull/161/files) was to have single repository, and in any way we will need duplicate Bean or methods, in that case I think it makes more sense now to go with methods instead of Bean. But we can also change that later, not in the scope of this pr.